### ISSUE TITLE
Invoking a method on a Blazor component?

### ISSUE THREAD

**Issue by NickDarvey** [fsbolero/Bolero#163]
Is there a way we can invoke a method on a Blazor component?

I have no idea how this would look with MVU, but I'm trying to use [BlazorDateRange ](https://github.com/jdtcn/BlazorDateRangePicker) in my project and you control opening and closing via methods, not setting an `IsOpen` value. I guess because it does a few things when setting the open state:
https://github.com/jdtcn/BlazorDateRangePicker/blob/f44426092ec17bd132d86353fa4d56e25844ad23/BlazorDateRangePicker/DatePickerComponentBase.cs#L333-L364

My thoughts for now was wrapping it in my own component which translates a value (`IsOpen`) to invocations of said methods. Is there a better way?

**Comment by Tarmil** [comment]
To do this in Blazor, you would use a `@ref` attribute to bind the component to a property in your own component.

```razor
<BlazorDateRange @ref="myDateRange" />

@code {
    BlazorDateRange myDateRange;

    void OpenMyDateRange()
    {
        myDateRange.Open();
    }
}
```

Right now in Bolero we have `ElementRefBinder` that can be used as an equivalent of `@ref` for an HTML element, but we don't have an equivalent for components. That's definitely a missing feature.

**Comment by NickDarvey** [comment]
Closed via #172
